toolkit/tools/scheduler/schedulerutils/printresults.go

```diff
@@ -5,14 +5,36 @@ package schedulerutils
 
 import (
 	"encoding/csv"
+	"fmt"
 	"os"
 	"path/filepath"
+	"sort"
 	"sync"
 
 	"github.com/microsoft/CBL-Mariner/toolkit/tools/internal/logger"
 	"github.com/microsoft/CBL-Mariner/toolkit/tools/internal/pkggraph"
+	"github.com/microsoft/CBL-Mariner/toolkit/tools/internal/sliceutils"
 )
 
+const (
+	colorReset = "\033[33m"
+	colorRed   = "\033[31m"
+	colorGreen = "\033[32m"
+	colorBlue  = "\033[34m"
+)
+
+func printInRed(inputString string) string {
```

Comment:
It would be goodness to have format versions of these functions.

Reply:
Are there any mature golang OSS packages that already provide good support for colorized output? That would be even better!